src/common/crypto/platform/openssl/crypto.cpp

```diff
@@ -281,6 +282,11 @@ ExpectedPrivateKey LoadFrom(const Args &args) {
 #endif // ndef MENDER_CRYPTO_OPENSSL_LEGACY
 
 ExpectedPrivateKey PrivateKey::Load(const Args &args) {
+	// Load OpenSSL config
+	if ((CONF_modules_load_file(NULL, NULL, 0) != 1)) {
+		log::Debug("Failed to load OpenSSL configuration file: " + GetOpenSSLErrorMessage());
```

Comment:
Shouldn't it be a warning? `Debug` messages won't be printed by default, and this message is very useful is the client later fails to authenticate for example. Or is it the conf file is not present in most of the environments?

Reply:
It's present, so I agree that warning is better.